Makefile

```diff
@@ -2,19 +2,18 @@ all: fmt check
 
 # Always keep the last released version here
 VERSION_REPLACES ?= v0.0.0
-VERSION ?= v0.0.1
+VERSION ?= v0.0.2
 export VERSION := $(VERSION)
 
 TARGET_NAMESPACE ?= kubevirt-hyperconverged
-export TARGET_NAMESPACE := $(TARGET_NAMESPACE)
-
 DEPLOY_DIR ?= manifests
 
 # Image registry variables
 QUAY_USER ?= $(USER)
 IMAGE_REGISTRY ?= quay.io/$(QUAY_USER)
-IMAGE_TAG ?= latest
+IMAGE_TAG ?= $(VERSION)
```

Comment:
Already updated. I wanted to test it with specific version.

Reply:
It's executed in `cluster/operator-push.sh:33`, but if local image with specific version will be build before locally, it will use it, but it could be confusing.